src/test/java/internship/framework/core/utility/TestUtils.java

```diff
@@ -0,0 +1,18 @@
+package internship.framework.core.utility;
+
+import internship.framework.pages.LoginPage;
+
+public class TestUtils {
+
+    private TestUtils() {
+    }
+
+    public static void login() {
+        LoginPage loginPage = new LoginPage().openPage();
+        loginPage.clickLogInButtonHome();
+        loginPage.enterEmail("stroeradu03@gmail.com")
```

Comment:
The method should have the option to set the email(username) and password. By doing so you are not constraint with a single case. e.g: test scenarios with different users.

Reply:
Fixed.